02-temperature_converter/temperature.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+//Importing scanner//
+public class temperature {
+
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    double Var = 1.8;
+    double Var2 = 32;
```

Comment:
do these need to be stored as variables?

Reply:
The reason I put them variables was because it made it easier for me. And I was not sure if I was gunna use them again that's also way.